### PR TITLE
adds card metadata support on payment

### PR DESCRIPTION
**Pull request by ryumer**

### CODE REVIEW

File: src/main/java/com/iyzipay/model/PaymentCard.java [hunk 1]
```diff
@@ -98,6 +109,7 @@ public String toString() {
                 .append("cardAlias", cardAlias)
                 .append("cardToken", cardToken)
                 .append("cardUserKey", cardUserKey)
+                .append("metadata", metadata)
```
Comment (sotuzun) [inline src/main/java/com/iyzipay/model/PaymentCard.java]: You should check payload string for map variables.

Comment (ryumer) [reply src/main/java/com/iyzipay/model/PaymentCard.java]: it has the format `metadata={}` for empty map and `metadata={kkey=vvalue}` for a singleton map

Comment (ryumer) [reply src/main/java/com/iyzipay/model/PaymentCard.java]: `metadata={kkey=vvalue, kkey2=vvalue2}` for full map =)

**Comment by sotuzun** [comment]
GitHub says branch is out of date.

**Pull request merged**